src/components/AddAccessory/toggle/ItemInfo/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as S from './ItemInfo.style';
+import { MODAL_DATA } from '../../../../constant/modalInfo';
+
+const ItemInfo = () => {
+  return (
+    <>
+      {MODAL_DATA.map((item) => (
+        <S.InfoWrapper key={item.id} $isLast={item.id === 5}>
+          <S.Icon>{item.icon}</S.Icon>
+          <S.Text>{item.text}</S.Text>
+        </S.InfoWrapper>
+      ))}
+    </>
+  );
```

Comment:
요 빈태그 없어두 될 것 같은데 왜 넣어줬을까용??! 요렇게 해도 되지 않나용!?
```suggestion
  return (
      {MODAL_DATA.map((item) => (
        <S.InfoWrapper key={item.id} $isLast={item.id === 5}>
          <S.Icon>{item.icon}</S.Icon>
          <S.Text>{item.text}</S.Text>
        </S.InfoWrapper>
      ))}
  );
```

Reply:
어 그러네요1! 삭제하고 머지하겠습니다!!